### PR TITLE
OP-245: Integrate ClaimsApp from Exact into Master version

### PR DESCRIPTION
**Pull request by druchniewicz**
Related ticket: https://openimis.atlassian.net/browse/OP-245

### CODE REVIEW

File: claimManagement/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -500,4 +500,47 @@
   <string name="empty_year">Please fill the year</string>
   <string name="DeleteFamilyOnlyOffline">Deleting of family will also Delete all Insurees, Policies and Premiums of this Family</string>
   <string name="LastInstallment">Last Installment</string>
+  <string name="title_activity_claim_review">ClaimReview</string>
```
Comment (dragos-dobre) [inline claimManagement/src/main/res/values/strings.xml:503]: I added these keys to Lokalize. Please update the demo french language also.

**Review by dragos-dobre — CHANGES_REQUESTED** [review]

File: claimManagement/src/main/java/org/openimis/imisclaims/ItemsAdapter.java [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+        public Reportmsg(final View itemView) {
+            super(itemView);
+
+
+            itemView.setOnClickListener(new View.OnClickListener() {
+                @Override
+                public void onClick(View view) {
+
+/*                    // Redraw the old selection and the new
+                    if(overViewItems.num.size() == 0){
```
[the remark this answers is not in the document]
Comment (druchniewicz) [reply claimManagement/src/main/java/org/openimis/imisclaims/ItemsAdapter.java]: Right, removed

File: claimManagement/src/main/java/org/openimis/imisclaims/ItemsAdapter.java [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+
+                    }*/
+
+                }
+            });
+
+            itemView.setOnLongClickListener(new View.OnLongClickListener() {
+                @Override
+                public boolean onLongClick(View view) {
+/*                    Context context = view.getContext();
```
[the remark this answers is not in the document]
Comment (druchniewicz) [reply claimManagement/src/main/java/org/openimis/imisclaims/ItemsAdapter.java]: Right, removed

File: claimManagement/src/main/res/values/strings.xml (hunk 1 above)
Comment (druchniewicz) [reply claimManagement/src/main/res/values/strings.xml]: Added

Comment (dragos-dobre) [reply claimManagement/src/main/res/values/strings.xml]: Are you sure you added all french translations? you can just dld the french strings file from Lokalize.

**Review by dragos-dobre — APPROVED** [review]

**Pull request merged**